### DISCUSSION TITLE
[已解决]请问有人收藏过20210720之前的mpv吗

### DISCUSSION THREAD

**Discussion by kli2013** [hooke007/mpv_PlayKit#226] [mpv 通用]
因为在使用webm脚本的时候使用hevc_nvenc显示开始编码了,但是进程一直不开始编码

从使用FFmpeg的时候发现了我用FFmpeg的新版会提示我更新驱动471.41,请求的api接口是11.1,但是我只有11

驱动是不可能更新的,所以我下载了471.41发布日期20210720之前的FFmpeg,成功使用了hevc_nvenc编码

但是在mpv的每日构建没找到21210720以前的,只保留了3页,最早是202112月的

有没有仓鼠玩家有保存啊?

**Comment by hooke007** [comment]
https://sourceforge.net/projects/mpv-player-windows/files/64bit/

**Comment by kli2013** [comment]
太久没去这个网站了,太卡了,一时间忘记在这里找了

测试旧版失败,应该是我显卡太老了,不但nevc_nvenc不支持,h264_nvnec也不支持
